app/src/main/java/com/willowtree/vocable/settings/EditCategoryOptionsFragment.kt

```diff
@@ -37,29 +36,27 @@ class EditCategoryOptionsFragment : BaseFragment<FragmentEditCategoryOptionsBind
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
 
-        val category = arguments?.getParcelable<Category>(KEY_CATEGORY)
+        // val category = arguments?.get(KEY_CATEGORY) as Category?
+        val category by lazy {
+            fromBundle(requireArguments()).category
```

Comment:
You can get around having to use `fromBundle` by using the `navArgs` extension.
``` kotlin
private val args: EditCategoryOptionsFragmentArgs by navArgs()
args.category
```

Reply:
Good catch!  Updated in both places.